tests/distributions/test_continuous.py

```diff
@@ -438,6 +438,20 @@ def test_exponential(self):
             {"lam": Rplus},
             lambda value, lam: st.expon.logcdf(value, 0, 1 / lam),
         )
+        check_icdf(
+            pm.Exponential,
+            {"lam": Rplus},
+            lambda q, lam: st.expon.ppf(q, loc=0, scale=1 / lam),
+        )
+        # Custom logp / logcdf / icdf check for invalid parameters
```

Comment:
This shouldn't be needed, the `check*` functions already do this. 

The reason we needed special logic for uniform functions was that the helper functions can't automatically figure out what is an invalid parameter, because it depends on both lower and higher. 

For these simpler cases the helper functions try `domain.lower-1` if you check the source code

Reply:
Yes this makes sense, thanks.
I've deleted the redundant checks and pushed a new commit.

However I pulled and rebased with the main branch and a new commit not authored by me is inserted here. Let me know if there is a better way to have done this.